sycl/include/sycl/detail/spirv.hpp

```diff
@@ -219,7 +220,8 @@ EnableIfGenericBroadcast<T> GroupBroadcast(T x, id<Dimensions> local_id) {
   if (Dimensions == 1) {
     return GroupBroadcast<Group>(x, local_id[0]);
   }
-  T Result;
+  // assign x to support type T without default constructor
```

Comment:
Likewise
```suggestion
  // Initialize with x to support type T without default constructor.
```

Reply:
Applied in https://github.com/intel/llvm/pull/7587/commits/f5881f95eab08e064f48332d4c54acd90a1004c9